deploy-scripts/deploy-scripts-constants.ts

```diff
@@ -0,0 +1,19 @@
+import { chainIds } from "../utils/Constants";
+import { TokenDeploymentData } from "./interfaces/deploy-scripts.interface";
+
+const deploymentScriptsFolder = "deploy-scripts";
+
+export const chainsDeployScriptsFolders = {
+  [chainIds.rsk]: [`${deploymentScriptsFolder}/rsk`],
+  [chainIds.mainnetFork]: [`${deploymentScriptsFolder}/rsk`],
```

Comment:
This is confusing (rsk value under mainnetFork), a comment would be appreciated.

Or maybe even rename the directory to "root-deployment" or "main-deployment"

Reply:
I would add a comment there, makes sense